website/src/content/concepts/oauth-scopes-and-user-permissions.mdx

```diff
@@ -30,6 +30,37 @@ Notice here how the OAuth Scopes are grouped by the two fields `view` and `manag
 
 This grouping determines the **mapping and relation between OAuth Scopes and user permissions**.
 
+# Additional OAuth Scopes
+
+Defining `oAuthScopes` in the Custom Application config allows using permissions limited to 1 unique pair (view/manage) specific to the Custom Application.
```

Comment:
Do you think it would help if we clarify the distinction between the default permission group and the additional groups?